pkg/admission/conf/am_conf.go

```diff
@@ -60,6 +61,9 @@ const (
 	AMAccessControlSystemUsers      = AccessControlPrefix + "systemUsers"
 	AMAccessControlExternalUsers    = AccessControlPrefix + "externalUsers"
 	AMAccessControlExternalGroups   = AccessControlPrefix + "externalGroups"
+
+	// label configurations
+	AMLabelDefaultQueue = LabelPrefix + "defaultQueue"
```

Comment:
Minor nit: Let's put these under filtering (ala `generateUniqueAppId`).

Reply:
Thanks for the review @craigcondit.
I think that looks better. Let me update the PR.

Reply:
@craigcondit, I've updated the PR. I'll resolve this conversation.